cobigen-setup/files/cliExecutionCommands.md

```diff
@@ -0,0 +1,18 @@
+CobiGen CLI is installed inside your devonfw distribution. In order to execute it follow the next steps:
+
+1. Run console.bat, this will open a console. Here we will go inside the devonfw ide installation to execute the commands.
+
+`cd root/devonfw`{{execute T1}}
+
+2. Execute cobigen or cg and the man page should be printed. Main entry point of the CLI. If no arguments are passed, man page will be printed.
+
+`cobigen`{{execute T1}}

```

Comment:
'cobigen' is not recognized as a command at this point. To make the command available in the terminal you have to execute '. ~/.bashrc' to refresh the bashrc file and make the command available. I would do this in the installCobigen step.